toqito/rand/random_circulant_gram.py

```diff
@@ -24,7 +24,8 @@ def random_circulant_gram(dim: int) -> np.ndarray:
     =========
     Generate a random circulant Gram matrix of dimension 4.
 
-    >>> from this_module import random_circulant_gram
+    >>> import numpy as np
+    >>> from toqito.rand import random_circulant_gram
```

Comment:
For all output in the `rand` module, some of the examples output a matrix. 

Is it Ok to make `doctest` skip testing these matrices? https://stackoverflow.com/a/7622355/10241324

Reply:
Hmm, I wonder if we should make a separate issue to return an `array` from `numpy` instead of a `matrix`. These functions that return a `matrix` object I think are kind of an older artifact of how `numpy` used to do things.